content/articles/recover-deleted-files-with-foremost/index.md

```diff
@@ -0,0 +1,121 @@
+### **Recovering deleted files using foremost**
+
+Unintentional deletion of important files can be tragic. This has led to operating systems having a restore point. In Linux, the restore point is the trash. In Windows, the restore point is the recycle bin. Sometimes we delete these files in the restore points too. At this point, we opt to visit data recovery centers or buy expensive data recovery software.
+
+### Introduction
+
+[**Foremost**](https://pkgs.org/download/foremost) is a free data retrieval software. You can download it on the internet. It operates on Linux operating systems. Throughout this tutorial, we will be using Kali Linux. Anyone using any other Linux distribution can follow through.
+
+In this tutorial, we shall be looking at the use of **foremost** in data recovery of lost/deleted files from a USB drive. We shall then go through how to recover deleted files from a whole hard disk. Finally, we will learn how to recover deleted files from a particular partition of a hard drive.
+
+### Prerequisites
+
```

Comment:
It is always advisable to include a Table of contents/Outline section. Please include one